test/jest/victory-chart/helper-methods.test.js

```diff
@@ -0,0 +1,48 @@
+import { getChildComponents } from "victory-chart/lib/helper-methods";
+import React from "react";
+import { VictoryAxis } from "victory-axis";
+import { VictoryLine } from "victory-line";
+import { Log } from "victory-core";
+
+describe("victory-chart/helpers-methods", () => {
+  const getVictoryLine = (props) => React.createElement(VictoryLine, props);
+  const getVictoryAxis = (props) => React.createElement(VictoryAxis, props);
+
+  describe("getChildComponents", () => {
+    const defaultAxes = {
+      independent: getVictoryAxis({}),
+      dependent: getVictoryAxis({ dependentAxis: true })
+    };
+    beforeEach(() => {
+      jest.spyOn(Log, "warn");
```

Comment:
Is this just to silence the logs during tests?

Reply:
I think so? I had just copied it over from the previous test, but I'll try removing it and see what happens.

Reply:
Removing it doesn't make a difference, so I'll just get rid of this.